util/src/bin/dns.rs

```diff
@@ -326,15 +326,15 @@ async fn tls(opts: Opts, provider: impl RuntimeProvider) -> Result<(), Box<dyn s
     Ok(())
 }
 
-#[cfg(not(feature = "__dns-over-https"))]
+#[cfg(not(feature = "__https"))]
 async fn https(
     _opts: Opts,
     _provider: impl RuntimeProvider,
 ) -> Result<(), Box<dyn std::error::Error>> {
-    panic!("`dns-over-https-rustls` feature is required during compilation");
+    panic!("`https-rustls` feature is required during compilation");
```

Comment:
This error message is out of sync with the config guard
```suggestion
    panic!("`__https` feature is required during compilation");
```